grouper/fe/forms.py

```diff
@@ -264,3 +265,12 @@ class UserTokenForm(Form):
     name = StringField("Token name", [
         validators.DataRequired(),
     ])
+
+
+class UserPasswordForm(Form):
+    name = StringField("Password name", [
```

Comment:
Is there a final decision on whether we want to restrict user input or just make them type the right name (if applications restrict what they use based on the password name)?